modules/tinymce/src/core/main/ts/undo/Setup.ts

```diff
@@ -12,6 +12,18 @@ import * as Levels from './Levels';
 import { endTyping, setTyping } from './TypingState';
 import { Locks, UndoLevel, UndoManager } from './UndoManagerTypes';
 
+const skipCommand = (cmd: string): boolean => {
+  switch (cmd) {
```

Comment:
```suggestion
  switch (cmd.toLowerCase()) {
```
This would be cleaner and would allow us to remove the `toLowerCase()` calls on lines 42 and 52.